main.py

```diff
@@ -63,7 +66,11 @@ def main():
 
         # Count the ratio to identify best next brewery to visit
         distances_df = pd.merge(
-            distances_df, geocodes_df.beer_count, left_index=True, right_index=True, how="inner"
+            distances_df,
+            geocodes_df.beer_count,
+            left_index=True,
+            right_index=True,
+            how="inner",
         )
```

Comment:
According to documentation pd.merge is used when you want to "merge" tables by values.
However, I see that in this case you areusing index in both sides.
That means that you should go with 
df.join(x) as this would join on index.
pd.concat([x1,x2]) is also a viable option as it join on index by default also